Alignment/OfflineValidation/test/eopTreeWriter_cfg.py

```diff
@@ -23,7 +23,7 @@
 
 # load configuration files
 process.load("Configuration.StandardSequences.MagneticField_cff")
-process.load("Configuration.Geometry.GeometryDB_cff")
+process.load("Configuration.StandardSequences.Geometry_cff")
```

Comment:
Hi @ianna, going back to this: I actually did not explicitly make this change, it must have happened as a result of an update which happened after I prepared the branch but before I created a PR (?).
I checked, and actually both "Configuration.Geometry.GeometryDB_cff" and 
"Configuration.StandardSequences.Geometry_cff" appear in the files. Namely:

bash-4.1$ while read file;do find $file -name "*" | xargs grep -l Configuration.Geometry.GeometryDB_cff; done < list_to_change
Alignment/OfflineValidation/python/TkAlAllInOneTool/alternateValidationTemplates.py
Alignment/OfflineValidation/python/TkAlAllInOneTool/geometryComparisonTemplates.py
Alignment/OfflineValidation/test/PVValidation_TEMPL_cfg.py
Alignment/OfflineValidation/test/dqmOfflinevalidator_cfg.py
Alignment/OfflineValidation/test/offlinevalidator_cfg.py
Alignment/OfflineValidation/test/testIntoNtuples_cfg.py
Alignment/OfflineValidation/test/testSplitterValidation_cfg.py

bash-4.1$ while read file;do find $file -name "*" | xargs grep -l Configuration.StandardSequences.Geometry_cff; done < list_to_change
Alignment/CommonAlignmentProducer/test/GlobalTrackerMuonAlignment_test_cfg.py
Alignment/MillePedeAlignmentAlgorithm/test/alignment_cfg.py
Alignment/MillePedeAlignmentAlgorithm/test/alignment_forGeomComp_cfg.py
Alignment/MillePedeAlignmentAlgorithm/test/alignment_survey_cfg.py
Alignment/MuonAlignmentAlgorithms/test/runDTLocalMillepede.py
Alignment/MuonAlignmentAlgorithms/test/runDTLocalMillepedeData.py
Alignment/MuonAlignmentAlgorithms/test/runDTLocalMillepedeFit.py
Alignment/OfflineValidation/python/TkAlAllInOneTool/alternateValidationTemplates.py
Alignment/OfflineValidation/test/eopTreeWriter_cfg.py

 Should I change both to what you suggest: "Configuration.Geometry.GeometryRecoDB_cff"? What would be the right thing to do